website/docs/Custom-Rules.md

```diff
@@ -2,7 +2,7 @@
 title: Custom Rules
 ---
 
-Rule file generation is a new and experimental feature of Premake 5.0, which currently only supports Visual Studio and the gmake2 action. It allows you describe how to build a particular kind of file, similar to [custom build commands](Custom-Build-Commands.md), but in a more generic way, and with variables that can be set in your project script.
+Rule file generation is a new and experimental feature of Premake 5.0, which currently only supports Visual Studio and the gmake action. It allows you describe how to build a particular kind of file, similar to [custom build commands](Custom-Build-Commands.md), but in a more generic way, and with variables that can be set in your project script.
```

Comment:
Unrelated to this PR, but "new and experimental" seems no longer be the case.
Supported by gmake/Codelite/vs*. only gmakelegacy and xcode doesn't support it.

Reply:
Yeah, we really need to do a deep scrub of the docs in the next few months.